src/main/scala/com/timushev/sbt/updates/UpdatesPlugin.scala

```diff
@@ -13,10 +13,11 @@ object UpdatesPlugin extends AutoPlugin {
   override val projectSettings = Seq(
     dependencyUpdatesReportFile := target.value / "dependency-updates.txt",
     dependencyUpdatesExclusions := DependencyFilter.fnToModuleFilter(_ => false),
+    dependencyUpdatesFilter := DependencyFilter.fnToModuleFilter(_ => true),
```

Comment:
I suppose there is some implicit conversion — for me `DependencyFilter.allPass` worked even though the setting type was `ModuleFilter`